samples/README.md

```diff
@@ -10,179 +10,34 @@ A State Machine for controlling an LED with a Raspberry Pi.
 
 A State Machine that replies to every message with an identical message.
 
-### Usage
-
-```
-start echo/Echo
-send <echo> test
-```
-
 ## Family
```

Comment:
In think these should be relative links to the associated README.md, I think this will work:
```
## [Family](family/README.md)
```

Reply:
Done: https://github.com/AletheiaWareLLC/Wink/pull/35